docs/src/api.md

```diff
@@ -0,0 +1,25 @@
+# APIs
+
+```@docs
+Quaternion
+```
+
+```@docs
+quat
```

Comment:
The docstring of `quat` will be added in #95.

Reply:
I added a temporary docstring in https://github.com/JuliaGeometry/Quaternions.jl/pull/97/commits/cffcb626b7ed3f8b74230f4ff071f9bd9cf29e21.